.secrets.baseline

```diff
@@ -0,0 +1,130 @@
+{
```

Comment:
is this file needed ? for every folder ? i thought its just needed for the root folder.